sdk/js/src/utils/consts.ts

```diff
@@ -324,8 +324,8 @@ const MAINNET = {
     nft_bridge: undefined,
   },
   xlayer: {
-    core: undefined,
-    token_bridge: undefined,
+    core: "0x194B123c5E96B9b2E49763619985790Dc241CAC0",
```

Comment:
Did you pass this info to Ben to make sure sdkV2 has parity?

Reply:
@barnjamin ^